test/e2e_node/services/kubelet.go

```diff
@@ -190,6 +197,58 @@ func createPodManifestDirectory() (string, error) {
 	return path, nil
 }
 
+// createKubeconfig creates a kubeconfig file at the fully qualified `path`.
+func createKubeconfig(path string) error {
+	kubeconfig := `apiVersion: v1
+kind: Config
+users:
+- name: kubelet
+clusters:
+- cluster:
+    server: ` + getAPIServerClientURL() + `
+    insecure-skip-tls-verify: true
```

Comment:
I think we're on the same page. My point was that changing things like this line won't disable unsecured localhost, we would have to do that somewhere else and see the things still using it start to fail.